BPOR/BPOR.Rms/Models/Researcher/ResearcherStudyFormViewModel.cs

```diff
@@ -83,9 +83,20 @@ public class GovUkDate
                 return null;
             }
         }
+        private static String? formatDateForUK(int? year, int? month, int? day)
+        {
+            if (!year.HasValue || !month.HasValue || !day.HasValue)
+            {
+                return null;
+            }
+
+            return ""+ day.Value + "/" + month.Value + "/" + year.Value;
+        }
 
         public DateOnly? ToDateOnly() => ConstructDate(Year, Month, Day);
 
+        public String? UKDisplayDate() => formatDateForUK(Year, Month, Day);
+
         public static GovUkDate FromDateTime(DateTime? date) => new GovUkDate { Day = date?.Day, Month = date?.Month, Year = date?.Year };
```

Comment:
@Tkirkwood Are we not able to just use this GovUkDate to display the dates in the UK format? Or does this still display in US format on the frontend?

Reply:
using the toDateOnly displays in theUS mm/dd/yyyy format.